--- configure.orig	2014-05-10 12:48:23.963872325 +0900
+++ configure	2014-05-10 12:48:10.781871546 +0900
@@ -4284,8 +4284,10 @@
 fi
 
 
-# Extract the first word of "llvm-config", so it can be a program name with args.
-set dummy llvm-config; ac_word=$2
+for ac_prog in llvm-config.x86 llvm-config
+do
+  # Extract the first word of "$ac_prog", so it can be a program name with args.
+set dummy $ac_prog; ac_word=$2
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for $ac_word" >&5
 $as_echo_n "checking for $ac_word... " >&6; }
 if ${ac_cv_path_LLVM_CONFIG+:} false; then :
@@ -4324,6 +4326,9 @@
 fi
 
 
+  test -n "$LLVM_CONFIG" && break
+done
+
 if test "x$LLVM_CONFIG" = "x"; then
   as_fn_error $? "llvm-config is not found." "$LINENO" 5
 fi
